src/mmw/js/src/modeling/gwlfe/entry/models.js

```diff
@@ -1,12 +1,73 @@
 "use strict";
 
-var Backbone = require('../../../../shim/backbone');
+var Backbone = require('../../../../shim/backbone'),
+    GwlfeModificationModel = require('../../models').GwlfeModificationModel;
+
+var EntryFieldModel = Backbone.Model.extend({
+    defaults: {
+        label: '',
+        name: '',
+        autoValue: null,
+        userValue: null,
+    },
+
+    initialize: function(attrs, options) {
+        if (options && options.toOutput) {
+            this.toOutput = options.toOutput;
+        }
+    },
+
+    toOutput: function() {
+        throw "Not implemented";
```

Comment:
Is this going to be implemented in the future? I think throwing here is probably fine, but this could also be a `console.warn` if we'd prefer not to have it cause an error.

Reply:
No, this will _always_ be overridden by the calculator's `toOutput`. If this is firing it means that it was not properly overridden. Not having this method here would've also raised an error at runtime if that were to happen, but I thought this made it clearer that this should over ridden. Maybe I can change the error output to say that explicitly.

Reply:
:+1: In that case I think `throw` is fine here since presumably if this ever got called but the calculation kept working the values would be incorrect anyway.